pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.32.0-dev6"
+__version__ = "0.32.0-dev7"
```

Comment:
```suggestion
__version__ = "0.32.0-dev8"
```

Since dev7 is taken by the PR, let's update this to there are no conflicts on merging to master.

Reply:
Yes, I know.